crypto/ec/ecp_nistputil.c

```diff
@@ -49,7 +49,8 @@
  * of size 'felem_size'. tmp_felems needs to point to a temporary array of
  * 'num'+1 field elements for storage of intermediate values.
  */
-void ec_GFp_nistp_points_make_affine_internal(size_t num, void *point_array,
+void
+ossl_ec_GFp_nistp_points_make_affine_internal(size_t num, void *point_array,
```

Comment:
I'd have broken the line at the `(` not the space after void.  It makes the argument list one or two lines.  Not critical.

Reply:
richard does not like the breaking at the (..